streaming/shared/prefix.py

```diff
@@ -128,7 +128,7 @@ def _check_and_find(streams_local: List[str], streams_remote: List[Union[str, No
                             f'Reused local directory: {streams_local} vs ' +
                             f'{their_locals}. Provide a different one. If using ' +
                             f'a unique local directory, try deleting the local directory and ' +
-                            f'call `streaming.base.util.clean_stale_shared_memory()` only once ' +
+                            f'call `streaming.util.clean_stale_shared_memory()` only once ' +
```

Comment:
This is a breaking change where if the user calls `streaming.base.util.clean_stale_shared_memory()`, it will fail since the `base` module no longer exists, and this API is being used outside. Is there a way where the user can call `streaming.base.util.clean_stale_shared_memory()` but will see a deprecation warning stating that the API path has been changed to `streaming.util.clean_stale_shared_memory()`?

Reply:
In the old code, this was not exported from either `streaming/__init__.py` or `streaming/base/__init__.py`, so I thought this is not covered by the no breaking changes rule?

In any case, I can mirror the old file tree, where each `base/` file just does ye olde `redirect(streaming/base/<path> -> streaming/<path>)`, for everything since it would be hard to justify only doing that one method?